benchmarks/benchmark_serving.py

```diff
@@ -676,6 +677,16 @@ def main(args: argparse.Namespace):
                 tokenizer=tokenizer,
                 return_prompt_formatted=True,
             )
+    elif args.dataset_name == "azure":
+        dataset = AzureDataset(
+            dataset_path=args.dataset_path,
```

Comment:
```suggestion
            random_seed=args.seed,
            dataset_path=args.dataset_path,
```

`random_seed` 값을 전달하는 부분이 누락된 것 같습니다.